### DISCUSSION TITLE
(MacOS): Two versions of mysql - neither work

### DISCUSSION THREAD

**Discussion by Vaska** [Homebrew/discussions#3439] [Getting started] [answered]
I've managed using brew for years with not much knowledge of things, but now I'm really stuck. Need some advice...

Updating, upgrading, etc., and mysql is broken - can not login, can not utilize it with internal web projects.

The mysql System Preferences panel says I am running 8.0.28. Brew, via the Terminal, says I am running 8.0.29. How would I go about resolving this? I don't care which version works...I just need one of them to be useful.

**Comment by SMillerDev** [comment]
> The mysql System Preferences panel says I am running 8.0.28.

That version was not installed through brew.

> Updating, upgrading, etc., and mysql is broken - can not login, can not utilize it with internal web projects.

Without knowing what broken looks like it is practically impossible to help you.

**Comment by Vaska** [comment, marked as answer]
I kept following the problem and came up with the idea of simply turning off the 8.0.28 version of mysql. Doing so - the Brew version of mysql started working again (I was able to run my internal server, for instance). In the end, this was probably not an actual issue with Brew.

Thanks for the advice.
